rhp/v3/rhp.go

```diff
@@ -478,4 +485,19 @@ type (
 		FailureRefund        types.Currency
 		Output               []byte
 	}
+
+	// RPCFinalizeProgramRequest is the finalization request object for the
+	// ExecuteProgram RPC.
+	RPCFinalizeProgramRequest struct {
+		Signature         []byte // siad encodes this field as a length-prefixed byte slice
```

Comment:
these can still be `types.Signature`, just encode as `e.WriteBytes(r.Signature[:])`. (This is already done for some other types, e.g. `PayByContractRequest`)